src/mudlet.cpp

```diff
@@ -527,6 +527,11 @@ mudlet::mudlet()
             return;
         }
 
+        host->mpEditorDialog = createMudletEditor();
+        if (!host->mpEditorDialog) {
+            qWarning() << "Failed to create editor dialog";
+            return;
+        }
```

Comment:
Given that the first of these two lines calls a method that first checks for the same `host` being non-null and then returns `host->mpEditorDialog` if itself is not null the logic here seems odd - also since `createMudletEditor()` itself assigns the address of the new `dlgTriggerEditor` **to** `host->mpEditorDialog` perhaps it would be more sensible to have:
```suggestion
        if (!host->mpEditorDialog && !createMudletEditor()) {
            qWarning() << "Failed to create editor dialog";
            return;
        }
```